selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -0,0 +1,65 @@
+from selenium.webdriver.common.by import By
+from selenium_tests.course_info.page_objects.course_info_base_page_object import CourseInfoBasePageObject
+from selenium.common.exceptions import NoSuchElementException
+
+
+class Locators(object):
+    ADD_PEOPLE_BUTTON = (By.XPATH, '//button[contains(.,"Add People")]')
+    ADD_PEOPLE_SEARCH_TXT = (By.ID, "emailHUID")
+    ADD_TO_COURSE_BUTTON = (By.ID, "add-user-btn-id")
+    ALERT_SUCCESS_PERSON = (By.XPATH, '//p[contains(.,"just added")]')
+    ROLES_DROPDOWN_LIST = (By.ID, "select-role-btn-id")
+
+    @classmethod
+    def TD_TEXT_XPATH(cls, search_text):
+        """ returns a locator for a table cell element in the people table;
+        search_text should be user's name, user_id, etc """
+        return By.XPATH, '//td[contains(text(), "{}")]'.format(search_text)
```

Comment:
My XPATH-fu is terrible.  If there's a way to scope this to be sure we're within the datatable, though, this would be a lot less fragile.  As is, we'll match any <TD> on the page with the search text in it, right?